style.css

```diff
@@ -81,3 +112,188 @@
 .social-pic li {
   list-style-type: none;
 }
+
+.card {
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  height: 552px;
+  width: 295px;
+  left: 24px;
+  top: 114px;
+  border-radius: 16px;
+  padding: 10px;
+  margin: auto;
+  margin-top: 100px;
+}
+
+.project-heading h2 {
+  width: 295px;
+  font-family: "Poppins", sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #172b4d;
+}
+
+.canopy {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 550;
+  font-size: 15px;
+  display: flex;
+  gap: 15px;
+  color: #7a869a;
+  margin-left: -40px;
+}
+
+.canopy li {
+  list-style-type: none;
+}
+
+.senior {
+  color: #172b4d;
+}
+
+.canopy-p {
+  width: 295px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+}
+
+.canopy-list {
+  display: flex;
+  gap: 25px;
+  font-family: 'Poppins', sans-serif;
+  margin-left: -40px;
+}
+
+.canopy-list li {
+  list-style-type: none;
+}
+
+.list {
+  font-size: 12px;
+  border: #000 solid 0;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-bottom: 5px;
+  padding-top: 8px;
+  border-radius: 8px;
+  background-color: #ebebff;
+  color: #6070ff;
+  height: 15px;
+}
+
+.list0 {
+  font-size: 12px;
+  font-family: "Poppins", sans-serif;
+  border: #000 solid 0;
+  padding: 12px;
+  border-radius: 8px;
+  background-color: #ebebff;
+  color: #6070ff;
+  height: 15px;
+}
+
+.card-img1 {
+  width: 295px;
+  border-radius: 8px;
+}
+
+.btn {
+  font-size: 17px;
+  padding: 10px;
+  border-radius: 8px;
+  background-color: #fff;
+  border: #6070ff solid 1px;
+  color: #396df2;
+  cursor: pointer;
+}
+
+.btn:hover {
+  color: #fff;
+  background-color: #396df2;
+}
+
+.about-section {
```

Comment:
- [ ]I can see that you tried to implement some of the requested changes in the previous review good job so far 👍 but there are some you didn't fix yet as requested in the previous review  kindly consider removing the border on the whole section and make sure that the border is on the right side only this will help your project to be a good duplicate of the provided design